src/components/content/condition-history/conditions-history.tsx

```diff
@@ -77,15 +54,15 @@ export function ConditionHistory({
         );
 
         const requestContext = await getRequestContext();
-        const provenanceBundles = await getProvenanceForConditions(
-          requestContext,
-          [condition, ...conditionsDataDeduped]
-        );
+        const provenances = await searchProvenances(requestContext, [
+          condition,
+          ...conditionsDataDeduped,
+        ]);
 
         let binaryId;
         const conditionsToDisplay = conditionsDataDeduped.map(
           (dedupdedCondition) => {
-            binaryId = getBinaryId(provenanceBundles, dedupdedCondition.id);
+            binaryId = getBinaryId(provenances, dedupdedCondition.id);
```

Comment:
Won't block on this, as the temporary closure created by `let binaryId;` was there previously. It makes more sense to just declare the variable inside the mapped function.
```suggestion
        const conditionsToDisplay = conditionsDataDeduped.map(
          (dedupdedCondition) => {
            const binaryId = getBinaryId(provenances, dedupdedCondition.id);
```

Reply:
Thanks! Done!